backend/app/src/test/kotlin/io/tolgee/api/v2/controllers/TranslationSuggestionControllerTest.kt

```diff
@@ -95,7 +89,8 @@ class TranslationSuggestionControllerTest : ProjectAuthControllerTest("/v2/proje
 
   @BeforeEach
   fun setup() {
-    mockCurrentDate { Date() }
+    setForcedDate(Date())
```

Comment:
This is probably what makes your tests fail. You have to clear the forced date in @AfterEach.

Reply:
That was indeed the problem! I'm feeling a bit silly for not noticing this on my own 😅